doc/decisions/DR014-Management-policy-responses.md

```diff
@@ -0,0 +1,256 @@
+file may have the trait set `{'file', 'image', 'raster',
+'colorManaged'}`.
+
+The `managementPolicy` method is a high-level (i.e. not entity specific)
+query that a host can use to determine a manager's behaviour in relation
+to a specific trait set.
+
+The answer to this query is commonly used by a Host to conditionally
+enable user-facing functionality based on the behaviour of the specific
+manager that is in use.
+
+This mechanism is flexible, and so exactly how a manager should
+implement its response is potentially ambiguous. This document outlines
+the rationale behind the defined canonical behaviour.
+
+## Relevant data
+
+Documentation pertaining to the OpenAssetIO data model:
+
+- [Entities, Traits and Specifications overview](../doxygen/src/EntitiesTraitsSpecifications.dox)
+- [Compositional data model decision record](./DR007-Hierarchical-or-compositional-traits-for-specifications.md)
+- [Unified Entity data model decision record](./DR008-Unify-the-entity-data-model.md)
+
+Notable points:
+
+- Trait sets combine multiple traits to increase specificity, rather
+  than meaning "or".
+- Some managers may only care about certain specific traits (such as
+  those describing the assets content disposition), and handle all
+  'derivative entity types' the same.
+- The `managementPolicy` query is not a required pre-requisite in the
+  use of `resolve`, et. al. but is used to adapt application behaviour
+  to improve user experience.
+
+## Illustrative scenario
+
+Consider a manager that pivots all behaviour around a
+`file` trait - that defines that any entity with that trait stores its
+data in a file. The manager's sole purpose is to manage the paths of
+such entities. It is not interested in any other kinds of entity, and
+should not be involved in their creation or management.
+
+As such, it wishes to opt-in to the management of any file-based entity,
+regardless of its specific trait set.
+
+The manager fulfills the API contract, by storing an entities trait set
+in its database - allowing it to filter lookups by trait set, and
+properly re-classify any given entity as required,. However, it can't
+store arbitrary data, and so can't persist the data for any other
+traits these entities may have.
+
+A Host that manages several file-based entities (eg: its
+main document and assorted data files that it reads/writes),
+will query the manager's `managementPolicy` to determine for which of
+these the manager should be involved with.
+
+If a manager opts-out of managing any given trait set, then the host
+will use its native UI/workflows for browsing and saving, if it opts-in,
+then it will delegate UI and data locality responsibilities to the
+manager.
+
+So, how should this manager respond to the `managementPolicy` query for
+any specific trait set?
+
+Lets explore the scenario where the host is querying the manager's
+policy for it's native document format, and for images it generates:
```

Comment:
s/it's/its/

Reply:
Fixed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/792/commits/59ec5fc906e84b5630641d7564614939a25f37cf) (with others).